package-lock.json

```diff
@@ -61625,7 +61625,7 @@
         "fb-watchman": "^2.0.0",
         "fbjs": "^3.0.0",
         "glob": "^7.1.1",
-        "immutable": "*",
+        "immutable": "~3.7.6",
```

Comment:
Is this change intentional? If not this file should be reverted. 

If you want to revert the file to its state in master in git:

```git checkout origin/master [filename]```

Reply:
It was not, I will do that now.